src/components/ResetPassword/ResetPassword.jsx

```diff
@@ -12,24 +12,26 @@ import {
   Heading,
   Stack,
   Box,
-  Link,
+  Grid,
+  GridItem,
+  useDisclosure,
+  InputRightElement,
+  IconButton,
+  InputGroup,
+  Text,
 } from '@chakra-ui/react';
+import { ViewIcon, ViewOffIcon, CheckIcon, CloseIcon } from '@chakra-ui/icons';
 import { confirmNewPassword } from '../../utils/AuthUtils';
-import styles from './ResetPassword.module.css';
-import { passwordRequirementsRegex } from '../../utils/utils';
+import PasswordConfirmationModal from '../PasswordConfirmationModal/PasswordConfirmationModal';
```

Comment:
🚫 **[eslint]** <[import/no-unresolved](https://github.com/import-js/eslint-plugin-import/blob/v2.27.5/docs/rules/no-unresolved.md)> <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>Unable to resolve path to module '../PasswordConfirmationModal/PasswordConfirmationModal'.